package.json

```diff
@@ -11,6 +11,7 @@
     "serve": "vue-cli-service serve",
     "build-bundle": "vue-cli-service build --target lib --name flow-builder src/lib.ts",
     "test:unit": "vue-cli-service test:unit",
+    "test:regenerate-storyshots": "vue-cli-service test:unit -u",
```

Comment:
Hi hi, any args we pass to `yarn test:regenerate-storyshots` will get forwarded onto the underlying call. Does that help? Like, if I did the following, the `vue-cli-service would receive the no-cache option:
```
$ yarn test:regenerate-storyshots --no-cache
```

With that in mind, do you think it would be fair to have just one command and we manually supply the additional flag when we need it?

Reply:
Aha, in that case, let's have `"vue-cli-service test:unit --no-cache"`, like what we have in travis config https://github.com/FLOIP/flow-builder/blob/master/.travis.yml#L8

`-u` shouldn't always be used for test

Reply:
Done!